async_simple/executors/test/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 file(GLOB executor_test_src "*.cpp")
 add_executable(async_simple_executor_test ${executor_test_src} ${PROJECT_SOURCE_DIR}/async_simple/test/dotest.cpp)
 
-target_link_libraries(async_simple_executor_test async_simple -pthread ${deplibs} ${testdeplibs})
+if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+    target_link_libraries(async_simple_executor_test async_simple ${deplibs} ${testdeplibs})
+else()
+    target_link_libraries(async_simple_executor_test async_simple -pthread ${deplibs} ${testdeplibs})
+endif()
```

Comment:
From the context, it looks like we could use ${CMAKE_THREAD_LIBS_INIT} here?

Reply:
No need change, revert is ok on windows, maybe i did not compile all projects cause such error before.